inc/classes/settings/class-secupress-settings.php

```diff
@@ -326,6 +332,8 @@ protected function do_sections() {
 		 */
 		do_action( 'secupress.settings.before_section_' . $this->sectionnow, $with_save_button );
 
+		$this->section_is_disabled = true;
```

Comment:
Not easy to understand if you don't have a global view indeed.
At the beginning I set it to "true", then I (maybe) set it to "false" later. And I set it to "null" at the end, just for the sake of resetting it.
To understand it all, I would say: "In a section, if all fields are disabled, then disable the submit button of this section".